src/utils/QCCollector.cpp

```diff
@@ -65,20 +65,22 @@ class OPENMS_DLLAPI QCCollector:
     registerInputFileList_("in_ProteinQuantifierProtein","<files>", StringList(), "Input files",false,false);
 	  registerInputFileList_("in_IDMapper","<files>", StringList(), "Input files",false,false);
 	  registerInputFileList_("in_MapRTTransformer","<files>", StringList(), "Input files",false,false);
-      registerInputFileList_("in_rawfiles_FalseDiscoveryRate","<files>", StringList(), "Input files",false,false);
+    registerInputFileList_("in_Internal_Calibration","<files>", StringList(), "Input files",false,false);
```

Comment:
lower/upper case is a bit mixed here...
OpenMS usually does everything lowercase with underscores